src/PAModel/Utility/Utilities.cs

```diff
@@ -328,5 +329,25 @@ public static void EnsurePathRooted(string path)
                 throw new ArgumentException("Path must be a full path: " + path);
             }
         }
+
+        /// <summary>
+        /// Relative path of the file depends on the fileType. eg: Image files go into "Assets\Images\"
```

Comment:
Can we infer this by file extension instead? (different directories is an unusual pattern for pro-devs)

Reply:
Yes, absolutely. We'll still need this function for the msapp side, but we could dump them flat in Assets/

Reply:
Discussed offline, keeping.